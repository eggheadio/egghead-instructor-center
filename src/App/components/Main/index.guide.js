import React from 'react'
import {storiesOf} from '@kadira/storybook'
import {lorem} from 'faker'
import Main from '.'

storiesOf('Main', module)
  .add('default', () => (
    <Main>
      {lorem.paragraphs()}
    </Main>
  ))
