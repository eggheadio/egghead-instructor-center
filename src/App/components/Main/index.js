import React, {PropTypes} from 'react'

const Main = ({children}) => (
  <main className='pa4 mw8 center'>
    {children}
  </main>
)

Main.propTypes = {
  children: PropTypes.node.isRequired,
}

export default Main
