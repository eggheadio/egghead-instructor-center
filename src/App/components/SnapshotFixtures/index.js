import React from 'react'

export const StringFixture = 'String'

export const ImageUrlFixture = 'https://s3.amazonaws.com/uifaces/faces/twitter/kimcool/128.jpg'

export const ComponentFixture = () => (
  <div>
    Component
  </div>
)
