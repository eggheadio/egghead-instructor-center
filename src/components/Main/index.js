import React, {PropTypes} from 'react'

const Main = ({children}) => (
  <main className='bg-tag-gray h-100 w-100 dark-gray ml7-ns mt4 mt0-ns'>
    <div className='mw8 pv4 ph3 ph4-ns'>
      {children}
    </div>
  </main>
)

Main.propTypes = {
  children: PropTypes.node.isRequired,
}

export default Main
