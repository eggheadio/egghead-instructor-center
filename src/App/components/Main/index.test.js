import React from 'react'
import {create} from 'react-test-renderer'
import {ComponentFixture} from '../SnapshotFixtures'
import Main from '.'

test('default', () => (
  expect(create(
    <Main>
      <ComponentFixture />
    </Main>
  )).toMatchSnapshot()
))
