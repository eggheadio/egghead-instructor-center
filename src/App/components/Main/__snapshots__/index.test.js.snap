exports[`test default 1`] = `
<main
  className="pa4 mw8 center">
  <div>
    Component
  </div>
</main>
`;
